src/test/ui/impl-trait/feature-gate-explicit_generic_args_with_impl_trait.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0632]: cannot provide explicit generic arguments when `impl Trait` is used in argument position
```

Comment:
...in fact, I imagine the longer for that part of the diagnostic will probably co-opt some of this logic